### PR TITLE
[EX] Edit database schema (2)

### PR DESCRIPTION
**Pull request by PKhing**
## What did you do

- remove `age`
- add `rejectedReason`

## Demo

-

### CODE REVIEW

**Review by miello — CHANGES_REQUESTED** [review]

**Comment by PKhing** [comment]
now actor9 and casting9 is pending
actor10 and casting10 is rejected
rejectedReason will be null if status is not rejected

File: packages/database/src/mocks/getBaseMock/index.ts [hunk 1]
```diff
@@ -51,23 +51,37 @@ export const getBaseMock = (
...
           '$2a$10$jikY3o7Apw/.NMTQjtLC1OMLUwEM73dpWaM7T5WEmjB5lguG/wZce',
-        type:
-          index <= NUMBER_OF_ADMIN
-            ? UserType.ADMIN
-            : index <= NUMBER_OF_ADMIN + NUMBER_OF_ACTOR
-            ? UserType.ACTOR
-            : UserType.CASTING,
-        status: UserStatus.ACCEPTED,
+        type,
+        status: getStatus(),
```
Comment (palmcm) [inline packages/database/src/mocks/getBaseMock/index.ts:82]:
```suggestion
        status,
```
why const status = getStatus() But not use

File: packages/database/src/mocks/getBaseMock/index.ts [hunk 2]
```diff
@@ -51,23 +51,37 @@ export const getBaseMock = (
         name: faker.name.firstName(),
       }
     case 'user':
+      const type =
+        index <= NUMBER_OF_ADMIN
+          ? UserType.ADMIN
+          : index <= NUMBER_OF_ADMIN + NUMBER_OF_ACTOR
+          ? UserType.ACTOR
+          : UserType.CASTING
```
Comment (palmcm) [inline packages/database/src/mocks/getBaseMock/index.ts:59]: Very readable, but ok

**Review by palmcm — COMMENTED** [review]
Wait for miello to Approve

**Review by miello — APPROVED** [review]

**Pull request merged**